main.py

```diff
@@ -1,11 +1,37 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
-
+    with open("players.json") as file:
+        data = json.load(file)
+    for name, data in data.items():
+        race, _ = Race.objects.get_or_create(
+            name=data.get("race").get("name"),
+            description=data.get("race").get("description")
+        )
+        guild = data.get("guild")
```

Comment:
The `guild` variable is being overwritten in line 18. Consider renaming the variable in line 16 to avoid confusion, or ensure that the logic correctly handles the case when `guild` is `None`.